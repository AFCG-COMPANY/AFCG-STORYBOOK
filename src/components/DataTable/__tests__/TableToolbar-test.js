import React from 'react';
import { mount } from 'enzyme';
import { TableToolbar } from '../';

describe('DataTable.TableToolbar', () => {
  it('should render', () => {
    const wrapper = mount(<TableToolbar className="custom-class" />);
    expect(wrapper).toMatchSnapshot();
  });
});
