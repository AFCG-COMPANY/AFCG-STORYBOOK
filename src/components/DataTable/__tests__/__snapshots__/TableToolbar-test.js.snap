// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.TableToolbar should render 1`] = `
<TableToolbar
  className="custom-class"
>
  <section
    className="bx--table-toolbar custom-class"
  />
</TableToolbar>
`;
